var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -38,9 +41,13 @@ class Podio(CMakePackage):
     depends_on('python', type=('build', 'run'))
     depends_on('py-pyyaml', type=('build', 'run'))
     depends_on('py-jinja2@2.10.1:', type=('build', 'run'), when='@0.12.0:')
+    depends_on('sio', type=('build', 'run'), when='@0.13.0: +sio')
```

Comment:
Are you implying that older versions of podio cannot build with sio support? In that case, you should add a `conflicts`.

Reply:
Older versions do not have sio support, but they should build just fine, because then they do not use anything from there. However, the newest version can use sio, if desired. That was the intention of adding it this way, but maybe I have missed the correct way to do it?

Reply:
In that case, I would still add:
```python
conflicts('+sio', when='@:0.12', msg='No sio support in older podio')
```
or something like that. We don't want `podio@0.12+sio` and `podio@0.12~sio` to be two different hashes/installations if they install the same thing.

Reply:
Right, that makes sense. Will change that and remove the `@0.13.0:` from here.